tools/cli/NearObjectCli.cxx

```diff
@@ -231,6 +231,42 @@ AddEnumOption(CLI::App* app, std::optional<EnumType>& assignTo, bool isMandatory
     }
 }
 
+/**
+ * @brief Displays and returns application configuration parameters.
+ *
+ * @param applicationConfigurationParameterData The possible application configuration parameters that may be set.
+ * @return std::vector<uwb::protocol::fira::UwbApplicationConfigurationParameter>.
+ */
+std::vector<uwb::protocol::fira::UwbApplicationConfigurationParameter>
+ProcessApplicationConfigurationParameters(UwbApplicationConfigurationParameterData applicationConfigurationParameterData)
+{
+    std::vector<uwb::protocol::fira::UwbApplicationConfigurationParameter> applicationConfigurationParameters;
+
+    std::cout << "Selected parameters:" << std::endl;
+    for (const auto& [applicationConfigurationParameter, applicationConfigurationParameterValue] : applicationConfigurationParameterData.GetValueMap()) {
```

Comment:
Could you put the value map into its own variable? The compiler is going to do this anyway, but having an explicit variable makes it far easier to inspect the values when running under a debugger:

```suggestion
    const auto applicationConfigurationParameterValues = applicationConfigurationParameterData.GetValueMap();
    for (const auto& [applicationConfigurationParameter, applicationConfigurationParameterValue] : applicationConfigurationParameterValues) {
```